docs/user/ovirt/install_upi.md

```diff
@@ -211,14 +327,34 @@ storage domains for masters and workers. This might be useful as etcd key-value
       template: worker_rhcos_tpl
       operating_system: "rhcos_x64"
       type: high_performance
-      graphical_console:
-        ...
       disks:
-        ...
+      - size: 120GiB
+        name: os
+        interface: virtio_scsi
+        storage_domain: depot_nvme
       nics:
-        ...
+      - name: nic1
+        network: lab
+        profile: lab
 ```
 
+The user can customize the parameters of both profiles according to the needs and the minimum requirements.
+
+* `cluster`: it's already set according to the value of the variable in the [General Section](#general-section).
+* `memory, sockets, cores`: mandatory parameters necessary to define the common specs of the VMs generated from this profile.
+* `template`: name of the template that the virtual machine will be based on (refer to [this](#create-templates-and-vms) for further information about templates).
+* `operating_system`: sets the vm OS type. With RHV 4.4 it's mandatory to use the value `rhcos_x64` to allow the `ignition script` 
```

Comment:
I think in upstream docs we should `s/RHV/oVirt/`

Reply:
Agree, I'll change it

Reply:
I'll change it from RHV to oVirt/RHV like it is also in the IPI docs